### PR TITLE
Remove redudant steps from instruction

### PR DESCRIPTION
**Pull request by hasanrashid**
This PR is intended to resolve [issue 40377](https://github.com/kubernetes/website/issues/40377)

Under the section **"Securing the service"** In [https://kubernetes.io/docs/tutorials/services/connect-applications-service/#securing-the-service](https://kubernetes.io/docs/tutorials/services/connect-applications-service/#securing-the-service):

1. A link to the default.conf in examples/staging/https-nginx/default.conf` is provided
2. Redundant steps  to create secrets and yml file are removed

### CODE REVIEW

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 1]
```diff
@@ -292,6 +292,8 @@ And also the configmap:
 ```shell
 kubectl create configmap nginxconfigmap --from-file=default.conf
 ```
+Note: the default.conf is located in [https://github.com/kubernetes/examples/tree/master/staging/https-nginx/](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/)
```
Comment (windsonsea) [inline content/en/docs/tutorials/services/connect-applications-service.md:295]:
```suggestion

Note: The `default.conf` file is located in
[https://github.com/kubernetes/examples/tree/master/staging/https-nginx/](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/).
```

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 2]
```diff
@@ -302,6 +304,47 @@ kubectl get configmaps
 NAME             DATA   AGE
 nginxconfigmap   1      114s
 ```
+
+Details of the configmap ```nginxconfigmap``` can be viewed with the following command
```
Comment (windsonsea) [inline content/en/docs/tutorials/services/connect-applications-service.md:308]:
```suggestion
You can view the details of the `nginxconfigmap` ConfigMap using the following command:

```
It's better to use an active statement to describe a command operation.

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 3]
```diff
@@ -302,6 +304,47 @@ kubectl get configmaps
...
+```
+kubectl describe cm nginxconfigmap
+```
+The output can look like the following
```
Comment (windsonsea) [inline content/en/docs/tutorials/services/connect-applications-service.md:312]:
```suggestion

The output is similar to:
```
This change is only for style consistency.

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 4]
```diff
@@ -302,6 +304,47 @@ kubectl get configmaps
 NAME             DATA   AGE
 nginxconfigmap   1      114s
 ```
+
+Details of the configmap ```nginxconfigmap``` can be viewed with the following command
+```
+kubectl describe cm nginxconfigmap
+```
```
Comment (windsonsea) [inline content/en/docs/tutorials/services/connect-applications-service.md:311]:
```suggestion
```shell
kubectl describe cm nginxconfigmap
```
```

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 5]
```diff
@@ -302,6 +304,47 @@ kubectl get configmaps
...
+The output can look like the following
+
+```
```
Comment (windsonsea) [inline content/en/docs/tutorials/services/connect-applications-service.md:314]:
```suggestion
```console
```

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 6]
```diff
@@ -292,6 +292,10 @@ And also the configmap:
 ```shell
 kubectl create configmap nginxconfigmap --from-file=default.conf
 ```
+
+Note: The `default.conf` file is located in
+[https://github.com/kubernetes/examples/tree/master/staging/https-nginx/](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/).
```
Comment (sftim) [inline content/en/docs/tutorials/services/connect-applications-service.md:297]: Try https://github.com/kubernetes/examples/tree/bc9ca4ca32bb28762ef216386934bef20f1f9930/staging/https-nginx/ (pinning to a commit)

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 7]
```diff
@@ -292,6 +292,10 @@ And also the configmap:
 ```shell
 kubectl create configmap nginxconfigmap --from-file=default.conf
 ```
+
+Note: The `default.conf` file is located in
```
Comment (sftim) [inline content/en/docs/tutorials/services/connect-applications-service.md:296]:
```suggestion
You can find an example for `default.conf` in
```

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 8]
```diff
@@ -302,6 +306,47 @@ kubectl get configmaps
 NAME             DATA   AGE
 nginxconfigmap   1      114s
 ```
+
+Details of the configmap ```nginxconfigmap``` can be viewed with the following command
+```shell
+kubectl describe cm nginxconfigmap
+
```
Comment (sftim) [inline content/en/docs/tutorials/services/connect-applications-service.md:313]: There's a line missing here. Also, write `configmap` not `cm` - it helps people learn.

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 9]
```diff
@@ -311,7 +356,7 @@ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /d/tmp/nginx.key -ou
 cat /d/tmp/nginx.crt | base64
 cat /d/tmp/nginx.key | base64
 ```
-
+<!--
```
Comment (sftim) [inline content/en/docs/tutorials/services/connect-applications-service.md:359]: Why comment this out?

File: content/en/docs/tutorials/services/connect-applications-service.md (hunk 8 above)
Comment (hasanrashid) [reply content/en/docs/tutorials/services/connect-applications-service.md:314]: Changed cm to conffigmap. Were you referring to a missing line after the kubectl describe cm nginxconfigmap
 line? That has been added as well

Comment (sftim) [reply content/en/docs/tutorials/services/connect-applications-service.md:314]: I'd expect to see <tt>```</tt> on this line, ending the sample command.

**Pull request merged**